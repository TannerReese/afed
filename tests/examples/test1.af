# Tests multiline comments
# Tests for null constant
# Tests boolean arithmetic
# Tests string arithmetic

this #{ Check that
	multiline comments,
can be placed anytwhere
}# : {
	x : [ true + false #{ Try bolean arithmetic }# =  `Before`
	,	"Hel" + "lo, " + "World!" # And string arithmetic
	= ``,
	null=``
]
}

