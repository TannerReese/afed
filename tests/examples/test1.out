# Tests multiline comments
# Tests for null constant
# Tests boolean arithmetic
# Tests string arithmetic

this #{ Check that
	multiline comments,
can be placed anytwhere
}# : {
	x : [ true + false #{ Try bolean arithmetic }# =  `true`
	,	"Hel" + "lo, " + "World!" # And string arithmetic
	= `"Hello, World!"`,
	null=`null`
]
}

